src/Command/ImportTimelineCommand.php

```diff
@@ -0,0 +1,320 @@
+
+    public function __construct(EntityManagerInterface $em, TimelineFactory $factory)
+    {
+        $this->em = $em;
+        $this->factory = $factory;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setName('app:import:timeline')
+            ->addArgument('profilesUrl', InputArgument::REQUIRED)
+            ->addArgument('themesUrl', InputArgument::REQUIRED)
+            ->addArgument('measuresUrl', InputArgument::REQUIRED)
+            ->setDescription('Import timeline from CSV files');
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->em->getConnection()->getConfiguration()->setSQLLogger(null);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $output->writeln(['', 'Starting Timeline import.']);
+
+        $this->em->beginTransaction();
+
+        $this->importProfiles($input, $output);
+        $this->importThemes($input, $output);
+        $this->importMeasures($input, $output);
+        $this->importFeaturedThemeMeasures($input, $output);
+
+        $this->em->commit();
+
+        $output->writeln(['', 'Timeline imported successfully!']);
+    }
+
+    private function importProfiles(InputInterface $input, OutputInterface $output): void
+    {
+        $profilesUrl = $input->getArgument('profilesUrl');
+
+        $output->writeln(['', sprintf('Starting profiles import from "%s".', $profilesUrl)]);
+
+        $count = 0;
+        foreach ($this->parseCSV($profilesUrl) as $index => $row) {
+            list($title, $description) = $row;
+
+            if (empty($title)) {
+                throw new \LogicException(sprintf('No title found for profile. (line %s)', $index+2));
+            }
+
+            if (empty($description)) {
+                throw new \LogicException(sprintf('No description found for profile "%s". (line %s)', $title, $index+2));
+            }
+
+            $this->em->persist($this->factory->createProfile($title, $description));
+
+            $count++;
+        }
+
+        $this->em->flush();
+        $this->em->clear();
+
+        $output->writeln(sprintf('Saved %s profiles.', $count));
+    }
+
+    private function importThemes(InputInterface $input, OutputInterface $output): void
+    {
+        $themesUrl = $input->getArgument('themesUrl');
+
+        $output->writeln(['', sprintf('Starting themes import from "%s".', $themesUrl)]);
+
+        $count = 0;
+        foreach ($this->parseCSV($themesUrl) as $index => $row) {
+            list($title, $isFeatured, $description, $imageUrl) = $row;
+
+            if (empty($title)) {
+                throw new \LogicException(sprintf('No title found for theme. (line %s)', $index+2));
+            }
+
+            $isFeatured = strtolower($isFeatured);
+            if (!array_key_exists($isFeatured, self::CHOICES_BOOLEAN)) {
+                throw new \LogicException(sprintf(
```

Comment:
`RuntimeException`, because the error comes from a passed (external) value.
A `LogicException` should be thrown only when a dev does a wrong usage of the API (i.e calling methods in a wrong order, using an object not initialized, ...).